src/client/graphics/layers/RadialMenu.ts

```diff
@@ -267,7 +267,36 @@ export class RadialMenu implements Layer {
   }
 
   tick() {
```

Comment:
(See comment below for details)
```suggestion
  async tick() {
```